code/__DEFINES/layers.dm

```diff
@@ -43,6 +43,7 @@
 #define WINDOW_FRAME_LAYER 2.72
 
 #define XENO_HIDING_LAYER 2.75
+#define PROJECTILE_HIT_THRESHHOLD_LAYER 2.76
```

Comment:
uh what

Reply:
/tg/ code, this is the threshold at which things that are under it won't get hit (unless you aim directly at them)

Reply:
@spookydonut is there anything wrong with this layer thingy or no? Should I remove it and use some other layer for the projectiles? If not, I consider this as resolved.

Reply:
Does this mean xenos hiding won't get hit by bullets that don't directly aim on them. But tables will?

Reply:
Guess we'll have to test.

Reply:
Fixed now.